samples/KafkaFlow.Sample/Program.cs

```diff
@@ -60,17 +51,31 @@ static void Main(string[] args)
                     )
             );
 
+            services.AddSingleton<PrintConsoleProducer>();
+
             var provider = services.BuildServiceProvider();
 
             var bus = provider.CreateKafkaBus();
 
             bus.StartAsync().GetAwaiter().GetResult();
 
+            var printConsole = provider.GetService<PrintConsoleProducer>();
             var consumerAcessor = provider.GetRequiredService<IConsumerAccessor>();
 
             while (true)
             {
-                var input = Console.ReadLine();
+                Console.Write("Number of messages to produce, Pause, Resume, or Exit:");
+                var input = Console.ReadLine().ToLower();
+
+                if (int.TryParse(input, out var count))
```

Comment:
You can put this ' if ' statement inside the switch :
```csharp
case var input when new Regex(@"\d+").IsMatch(input):
```

Reply:
Changed to something similiar